_posts/2017-11-21-release-1.4-and-1.5-timeline.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title:  "Looking Ahead to Apache Flink 1.4.0 and 1.5.0"
+date:   2017-11-21 9:00:00
+categories: news
+authors:
+- stephan:
+  name: "Stephan Ewen"
+  twitter: "StephanEwen"
+- aljoscha:
+  name: "Aljoscha Krettek"
+  twitter: "aljoscha"
+- mike:
+  name: "Mike Winters"
+  twitter: "wints"
+---
+
+The Apache Flink 1.4.0 release is on track to happen in the next couple of weeks, and for all of the
+readers out there who haven’t been following the release discussion on [Flink’s developer mailing
+list](http://flink.apache.org/community.html#mailing-lists), we’d like to provide some details on
+what’s coming in Flink 1.4.0 as well as a preview of what the Flink community will save for 1.5.0.
+
+Both releases include ambitious features that we believe will move Flink to an entirely new level in
+terms of the types of problems it can solve and applications it can support. The community deserves
+lots of credit for its hard work over the past few months, and we’re excited to see these features
+in the hands of users.
+
+This post will describe how the community plans to get there and the rationale behind the approach.
+
+## Coming soon: Major Changes to the Flink’s Runtime
+
+There are 3 significant improvements to the Apache Flink engine that the community has nearly
+completed and that will have a meaningful impact on Flink’s operability and performance.
+
+1. Rework of the deployment model and distributed process
+2. Transition from configurable, fixed-interval network I/O to event-driven network I/O and application-level flow control for better backpressure handling
+3. Faster recovery from failure
+
+Next, we’ll go through each of these improvements in more detail.
+
+## Reworking Flink’s Deployment Model and Distributed Process
```

Comment:
"Distributed Processing"?

Reply:
@wints what do you think?

Reply:
+1 to changing to "distributed processing"